Source/santad/SNTExecutionController.m

```diff
@@ -78,17 +80,45 @@ - (instancetype)initWithEventProvider:(id<SNTEventProvider>)eventProvider
     // This establishes the XPC connection between libsecurity and syspolicyd.
     // Not doing this causes a deadlock as establishing this link goes through xpcproxy.
     (void)[[MOLCodesignChecker alloc] initWithSelf];
+
+    SNTMetricSet *metricSet = [SNTMetricSet sharedInstance];
+    _events = [metricSet counterWithName:@"/santa/events"
+                              fieldNames:@[ @"action_response" ]
+                                helpText:@"Events processed by Santa with response"];
```

Comment:
nit: "Events processed by Santa per response"

Reply:
Yup. I've changed it.